MapboxAndroidDemo/build.gradle

```diff
@@ -2,83 +2,84 @@ apply plugin: 'com.android.application'
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
+
+    defaultConfig {
+        applicationId "com.mapbox.mapboxandroiddemo"
+        minSdkVersion 15
+        targetSdkVersion 25
+        versionCode gitVersionCode
+        versionName gitVersionName
     }
 
-    nogpservices {
-    }
-  }
-
-  buildTypes {
-    debug {
-      applicationIdSuffix '.debug'
-      versionNameSuffix '-Debug'
-      minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    productFlavors {
+        gpservices {
+            resValue "string", "firebase_app_id", "1:557332920931:android:bdf11618ae45dd61"
+            resValue "string", "firebase_api_key", "AIzaSyCGCYLyLoi31Hv-pkBO0D51zCyxjngJbz8"
+        }
+
+        nogpservices {
+        }
     }
 
-    release {
-      minifyEnabled true
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    buildTypes {
+        debug {
+            applicationIdSuffix '.debug'
+            versionNameSuffix '-Debug'
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+
+        release {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
     }
-  }
 }
 
 repositories {
-  maven { url "https://jitpack.io" }
-  maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
+    maven { url "https://jitpack.io" }
+    maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
 }
 
 dependencies {
 
-  // Wearable application module
-  gpservicesWearApp project(':MapboxAndroidWearDemo')
-
-  compile "com.android.support:gridlayout-v7:${supportLibVersion}"
-  compile "com.android.support:appcompat-v7:${supportLibVersion}"
-  compile "com.android.support:cardview-v7:${supportLibVersion}"
-  compile "com.android.support:recyclerview-v7:${supportLibVersion}"
-  compile "com.android.support:design:${supportLibVersion}"
-  compile 'com.github.javiersantos:MaterialStyledDialogs:2.1'
-  compile 'com.getbase:floatingactionbutton:1.10.1'
-  compile 'com.squareup.picasso:picasso:2.5.2'
+    // Wearable application module
+    gpservicesWearApp project(':MapboxAndroidWearDemo')
 
-  // firebase crash report
-  gpservicesCompile 'com.google.firebase:firebase-crash:10.2.0'
+    compile "com.android.support:gridlayout-v7:${supportLibVersion}"
+    compile "com.android.support:appcompat-v7:${supportLibVersion}"
+    compile "com.android.support:cardview-v7:${supportLibVersion}"
+    compile "com.android.support:recyclerview-v7:${supportLibVersion}"
+    compile "com.android.support:design:${supportLibVersion}"
 
-  // Mapbox dependencies
-  compile ('com.mapbox.mapboxsdk:mapbox-android-sdk:5.1.0-SNAPSHOT@aar') {
-    transitive = true
-  }
+    // firebase crash report
```

Comment:
Looks like things got a bit out of order it this file? If nothing has changed in the `build.gradle` maybe just remove the file from this PR?